pkg/handler/v2/external_cluster/aks.go

```diff
@@ -150,6 +213,41 @@ func ListAKSClustersEndpoint(userInfoGetter provider.UserInfoGetter, projectProv
 	}
 }
 
+func ListAKSVMSizesEndpoint(presetProvider provider.PresetProvider, userInfoGetter provider.UserInfoGetter) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(AKSVMSizesReq)
```

Comment:
I suggest checking if casting works as expected in all such places:
```suggestion
		req, ok := request.(AKSVMSizesReq)
		if !ok {...}
```